WalletWasabi.Fluent/Views/CoinJoinProfiles/CoinJoinProfilesView.axaml

```diff
@@ -26,10 +26,13 @@
               Content="Customize" />
 
       <ListBox Items="{Binding Profiles}" SelectedItem="{Binding SelectedProfile}" Background="Transparent"
+               VerticalAlignment="Center"
                ScrollViewer.VerticalScrollBarVisibility="Disabled">
         <ListBox.Styles>
           <Style Selector="ListBoxItem">
             <Setter Property="Cursor" Value="Hand" />
+            <Setter Property="Margin" Value="15 20" />
```

Comment:
There is still a little edge outside of the items that can be clicked.
Removing the padding and increasing the margin can fix it.

```suggestion
            <Setter Property="Margin" Value="27 32" />
            <Setter Property="Padding" Value="0" />
```